### PR TITLE
Timeline section

### PR DESCRIPTION
**Pull request by demiwu96**
delete extra images and put social media links on footer

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -201,14 +215,14 @@ export default function Footer() {
...
-                            <a href="#">
+                            <a href="https://www.instagram.com/civictechto/?hl=en">
                                 <Avatar className={classes.iconAvatar}>
                                     <InstagramIcon />
                                 </Avatar>
                             </a>
                         </li>
                         <li className={classes.iconLi}>
-                            <a href="https://twitter.com/?lang=en">
+                            <a href="https://twitter.com/CivicTechTO?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor">
```
Comment (hanlinc27) [inline src/components/Footer/Footer.js]: Can this be changed to : [https://twitter.com/CivicTechTO](https://twitter.com/CivicTechTO)

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -201,14 +215,14 @@ export default function Footer() {
                             </a>
                         </li>
                         <li className={classes.iconLi}>
-                            <a href="#">
+                            <a href="https://www.instagram.com/civictechto/?hl=en">
```
Comment (hanlinc27) [inline src/components/Footer/Footer.js]: [https://www.instagram.com/civictechto/](https://www.instagram.com/civictechto/)

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (demiwu96) [reply src/components/Footer/Footer.js]: fixed

File: src/components/Footer/Footer.js (hunk 2 above)
Comment (demiwu96) [reply src/components/Footer/Footer.js]: fixed

**Review by hanlinc27 — DISMISSED** [review]

**Comment by harshvchawla** [comment]
Thanks for including me on the review process @hanlinc27 
There are some suggestions I would like to be integrated if possible in this merge
Will reply more on tmrow on this (mostly related to best practices around keeping hard-coded values away from production files, and trying to find a balance between hardcode and dynamic variables)

**Comment by rishFilet** [comment]
> Thanks for including me on the review process @hanlinc27
> There are some suggestions I would like to be integrated if possible in this merge
> Will reply more on tmrow on this (mostly related to best practices around keeping hard-coded values away from production files, and trying to find a balance between hardcode and dynamic variables)

What about config files?

**Review by rishFilet — DISMISSED** [review]

**Review by rishFilet — APPROVED** [review]

**Review by hanlinc27 — APPROVED** [review]

**Comment by harshvchawla** [comment]
I think we can tackle the refactoring (dynamic values etc) separately and can merge this pull request when others have already approved

**Review by harshvchawla — APPROVED** [review]

**Pull request merged**